apps/condo/domains/organization/schema/InviteNewOrganizationEmployeeService.js

```diff
@@ -17,9 +17,22 @@ const { Organization, OrganizationEmployee, OrganizationEmployeeSpecialization,
 const guards = require('@condo/domains/organization/utils/serverSchema/guards')
 const { PHONE_TYPE, EMAIL_TYPE } = require('@condo/domains/user/constants/identifiers')
 const { createUser } = require('@condo/domains/user/utils/serverSchema')
+const { getIdentificationUserRequiredFields } = require('@condo/domains/user/utils/serverSchema/userHelpers')
+
 
 const INVITE_REQUIRE_ALLOWED_FIELDS = [PHONE_TYPE, EMAIL_TYPE]
-const INVITE_REQUIRED_FIELDS = conf['INVITE_REQUIRED_FIELDS'] ? JSON.parse(conf['INVITE_REQUIRED_FIELDS']) : [PHONE_TYPE]
+let INVITE_REQUIRED_FIELDS = conf['INVITE_REQUIRED_FIELDS'] ? JSON.parse(conf['INVITE_REQUIRED_FIELDS']) : [PHONE_TYPE]
+
+
+if (conf.IDENTIFICATION_USER_REQUIRED_FIELDS !== undefined && conf.INVITE_REQUIRED_FIELDS !== undefined) {
```

Comment:
Let's leave TODO to remove INVITE_REQUIRED_FIELDS env

Reply:
added task